apps/condo/domains/user/components/auth/ValidatePhoneForm.css

```diff
@@ -0,0 +1,9 @@
+.secondary-link a.condo-typography {
```

Comment:
why not default link behavior?

Reply:
Our links from UI kit have only one color